frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -244,6 +245,21 @@ const getAnnotation = (obj: K8sResourceKind, annotation: string): string => {
   return _.get(obj, ['metadata', 'annotations', annotation]);
 };
 
+export const mapAnnotationsToJSON = (
+  annotations: ObjectMetadata['annotations'],
+  annotationKeys: string[],
+  onError = (error: Error, key?: string, value?: string) => console.warn(error)): any[] => {
```

Comment:
We should print the annotation key with the error. I don't think key and value should be optional here. That means we might not pass a value when invoking the callback, which isn't the case here.

```suggestion
  onError = (error: Error, key: string, value: string) => console.warn(`Could not parse annotation ${key} as JSON:`, error)): any[] => {
```

Reply:
Fixed